source/opt/dead_branch_elim_pass.h

```diff
@@ -111,9 +114,22 @@ class DeadBranchElimPass : public MemPass {
   // Return true if all extensions in this module are allowed by this pass.
   bool AllExtensionsSupported() const;
 
+  // Save next available id into |module|.
+  inline void FinalizeNextId(ir::Module* module) {
```

Comment:
I could also push this into (the questionably named) MemPass. I had been holding off because a few of the classes deriving from MemPass didn't use these. This *was* one, but now I am thinking of just doing it.

Reply:
I will do the move in a subsequent PR.

Reply:
I appreciate that.  There's lots of refactoring opportunity.  :-)